river/Cursor.zig

```diff
@@ -192,6 +203,11 @@ pub fn setTheme(self: *Self, theme: ?[*:0]const u8, _size: ?u32) !void {
     }
 }
 
+/// Hides the cursor.
+pub fn hide(self: *Self) void {
+    self.wlr_cursor.setImage(0, 0, 0, 0, 0, 0, 0);
+}
+
```

Comment:
I think we would need to save whatever the current cursor image is to be able to properly restore it on the next pointer input. Cursor images may be set by clients.

Leaving this feature for a second PR would likely get this one merged more quickly.